lib/generators/react_on_rails/templates/base/base/package.json

```diff
@@ -29,5 +29,6 @@
     "url": "https://github.com/shakacode/react-webpack-rails-tutorial/issues"
   },
   "homepage": "https://github.com/shakacode/react-webpack-rails-tutorial",
-  "dependencies": {}
+  "dependencies": {},
+  "cacheDirectories": ["node_modules", "client/node_modules"]
```

Comment:
it's possible the app has one for some other reason (separate service, `browserify` integration, etc) and it's the heroku default if nothing else set so i figured it made sense to keep it just in case.